osu.Game.Rulesets.Taiko/UI/DrumTouchInputArea.cs

```diff
@@ -142,15 +154,58 @@ private void handleUp(object source)
         private bool validMouse(MouseButtonEvent e) =>
             leftRim.Contains(e.ScreenSpaceMouseDownPosition) || rightRim.Contains(e.ScreenSpaceMouseDownPosition);
 
-        private TaikoAction getTaikoActionFromInput(Vector2 inputPosition)
+        #pragma warning disable format
```

Comment:
Please stick to the formatting guidelines, no need to disable it to fit with this style. This whole switch-within-a-switch isn't great to read imo, but I can't suggest anything overly better really